### DISCUSSION TITLE
Non identified phases

### DISCUSSION THREAD

**Discussion by Sofianehmz** [mtex-toolbox/mtex#1632] [Ask Anything] [answered]
Hello MTEX community, 
Hope you all doing good,

I'm a new user and I'm trying to plot the EBSD map. When I upload the data it shows that all the points have been taken as not indexed (as shown in the summary of the data to be imported in the image) and we can see that there are 2 phases from my EBSD data ( I have attached a copy) 
It has to do with the phase ID ? 
Thanks in advance 
![erreur ](https://user-images.githubusercontent.com/117661710/200343496-bfbc5ecb-48fc-47e7-9a36-340e87876130.png)
[MTEX1.txt](https://github.com/mtex-toolbox/mtex/files/9953123/MTEX1.txt)

**Comment by ralfHielscher** [comment, marked as answer]
You can use the following commands to import your data:
```matlab
CSList = {'noIndexed',...
  crystalSymmetry('432',[2.866,2.866,2.866],'mineral','Ferrite bcc'),...
  crystalSymmetry('432',[3.66,3.66,3.66],'mineral','Austenite, fcc')};
	
ebsd = EBSD.load('MTEX1.txt','columnNames',...
  {'Phase' 'X' 'Y' 'Bands' 'Error' 'Euler1' 'Euler2' 'Euler3' 'MAD' 'BC' 'BS'},...
  'CS',CSList)
```
Ralf.

**Reply by Sofianehmz** [reply]
Thank you so much Ralf
